Basic-Scripts/magic_square1.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 18 11:24:15 2020
+
+@author: Gulshan
+"""
+'''
+
+What is Magic Square?
+=> a square divided into smaller squares each containing a number,
+such that the figures in each vertical, horizontal, and diagonal row add up to the same value.
+
+Example:
+    2 7 6 
+    9 5 1 
+    4 3 8 
+    The sum of each row/column/diagonal is: 15.0
+    
+'''
+
+# Function to create Magic Square
+def magic_square(n):

```

Comment:
Change file_name to magic_square
Include docstrings for function documentation.
Use meaningful variable names